libexec/Makefile

```diff
@@ -44,7 +44,6 @@ _blacklistd-helper+=	blacklistd-helper
 SUBDIR+=	bootpd
 .endif
 
-# Not ported to RISCV yet
 .if !defined(NO_PIC) && !defined(NO_RTLD) && (${MACHINE_ABI:Mpurecap} || ${MK_COMPAT_CHERIABI} != "no") && !${MACHINE_ARCH:Mriscv*}
```

Comment:
```suggestion
.if !defined(NO_PIC) && !defined(NO_RTLD) && (${MACHINE_ABI:Mpurecap} || ${MK_COMPAT_CHERIABI} != "no")
```

Reply:
Oops, yes, managed to lose that when splitting up and reordering commits to extract out the earlier cleanup.